BraintreeDropIn/Public/BTDropInRequest.h

```diff
@@ -2,36 +2,22 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class BTPostalAddress;
+@class BTPostalAddress, BTPayPalRequest;
 
 @interface BTDropInRequest : NSObject <NSCopying>
 
-/// Optional: Amount of the transaction.
+/// Optional: Amount of the transaction for ThreeDSecure flows.
 ///
 /// Amount must be a non-negative number, may optionally contain exactly 2 decimal places
 /// separated by '.', optional thousands separator ',', limited to 7 digits before the decimal point.
 ///
-/// Used by PayPal single payments and ThreeDSecure.
+/// Note: You must still set the amount on the `payPalRequest` for PayPal One Time flows.
 @property (nonatomic, copy, nullable) NSString *amount;
 
-/// Optional: A valid ISO currency code to use for the transaction. Defaults to merchant currency code if not set.
+/// Optional: Specify the options for the PayPal flow. If not present, a default Billing Agreement flow will be used.
 ///
-/// Used by PayPal.
-@property (nonatomic, copy, nullable) NSString *currencyCode;
```

Comment:
Yeah, major version bumping. We never actually used this property so the impact will be superficial (anyone using it will need to remove the references).